ambry-router/src/test/java/com.github.ambry.router/GetBlobInfoOperationTest.java

```diff
@@ -149,7 +154,7 @@ public void onCompletion(BlobInfo result, Exception exception) {
     // test a good case
     GetBlobInfoOperation op =
         new GetBlobInfoOperation(routerConfig, routerMetrics, mockClusterMap, responseHandler, blobIdStr,
-            operationFuture, operationCallback, time);
+            operationFuture, operationCallback, operationCompleteCallback, time);
```

Comment:
In this case only actually. For the other one, the object isn't even constructed so that's ok.